index.html

```diff
@@ -328,13 +328,52 @@
 
       // Delete functionality:
 
+      function get_points(polyline) {
```

Comment:
Polyline with several points does not seem to work entirely, as they are connected two and two, and only able to delete parts of the segmented line at a time, as you mention.

Reply:
> does not seem to work entirely

Does not work at all, as it's not implemented ;) Note that the PR says preparation for line segments.